_data/apis.yml

```diff
@@ -10,13 +11,12 @@
     - gaming
   stock_api_domains_urls:
     - https://www.aisa.digital/e-commerce-translation/
-
   glossary: true
   self-serve: false
 
-
 - name: Alexa Translations A.I.
   id: alexa-ai
+  company: alexa-translations-ai
```

Comment:
In this case, probably the company id is correct, but the API id should also be like that.

Reply:
Fixed